sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/CustomEndpointUtils.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.translation.text;
+
+/** Utilities to check the Translator Custom endpoint. */
+public final class CustomEndpointUtils {
```

Comment:
Does this utility class need to be in the public API?

Reply:
Done